File: go/1174.go
package main

import "fmt"

func main() {
	var arr []float64
	var x float64

	for i := 0; i < 100; i++ {
		fmt.Scanf("%f", &x)
		arr = append(arr, x)
	}

	for i, v := range arr {
		if v > 10 {
			continue
		}
		fmt.Printf("A[%d] = %.1f\n", i, v)
	}
}

File: c/1174.c
#include <stdio.h>

int main ()
{

  float n, x[100];
  int i;

  for (i = 0; i < 100; i++)
  {
    scanf("%f", &n);
    x[i] = n;

  }

  for (i = 0; i < 100; i++)
  {

    if (x[i] <= 10)
    {

      printf("A[%d] = %.1f\n", i, x[i]);

    }
  }
}